st2mistral/tests/unit/test_jinja_render_complex.py

```diff
@@ -0,0 +1,31 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+from st2mistral.tests.unit import test_filter_base as base
+
+
+class JinjaUtilsToComplexFilterTestCase(base.JinjaFilterTestCase):
+
+    def test_filter_to_complex(self):
+        env = self.get_jinja_environment()
+        obj = {'a': 'b', 'c': {'d': 'e', 'f': 1, 'g': True}}
+
+        template = '{{k1 | to_complex}}'
```

Comment:
As discussed in https://github.com/StackStorm/st2/pull/3565, we're deciding (for now) to not support the filter format due to the upstream mistral issue, and requiring that users use the function format. I added `context` as the first positional arg (and modified testing accordingly) in https://github.com/StackStorm/st2mistral/pull/30/commits/739fa88157a218c4e56acae9aed1457c373269de